src/config.ts

```diff
@@ -488,7 +488,7 @@ const CHAINS: Chain[] = [
     rpc: rpcUrl
       .default("https://rpc-mumbai.maticvigil.com/")
       .parse(process.env.POLYGON_MUMBAI_RPC_URL),
-    pricesFromTimestamp: Date.UTC(2023, 9, 19, 0, 0, 0),
+    pricesFromTimestamp: Date.UTC(2023, 8, 19, 0, 0, 0),
```

Comment:
@bard I just noticed these changes are in Polygon, should they be here?

Reply:
Yes, I noticed the date was set in the future there so I fixed it.